src/tests/ie_test_utils/common_test_utils/graph_comparator.cpp

```diff
@@ -504,6 +504,17 @@ class CompareSubGraphs {
             }
         }
 
+        const auto lhs_body = sub_lhs->get_function();
+        const auto rhs_body = sub_rhs->get_function();
+        if (lhs_body && rhs_body) {
+            auto cmpr = FunctionsComparator::no_default();
+            cmpr.enable(FunctionsComparator::NODES);
```

Comment:
why is it not default? I believe we should inherit all settings from "main" graph comparator obj?
is it possible to re-use it and do not create a new one?

Reply:
Done - flags are inherited now.